plugins/modules/ecs_taskdefinition.py

```diff
@@ -277,7 +277,7 @@ def register_task(self, family, task_role_arn, execution_role_arn, network_mode,
         try:
             response = self.ecs.register_task_definition(**params)
         except botocore.exceptions.ClientError as e:
-            self.module.fail_json(msg=e.message, **camel_dict_to_snake_dict(e.response))
+            self.module.fail_json(msg=to_native(e), **camel_dict_to_snake_dict(e.response))
```

Comment:
This module is using AnsibleAWSModule, which means we have a 'fail' method which is much easier to use and avoids the need to for the extra import.

```suggestion
            self.module.fail_json_aws(e, msg="Failed to register task")
```